app/src/main/java/org/fossasia/openevent/app/common/data/models/CreateEvent.java

```diff
@@ -0,0 +1,113 @@
+package org.fossasia.openevent.app.common.data.models;
+
+import android.databinding.ObservableBoolean;
+
+import com.fasterxml.jackson.annotation.JsonAutoDetect;
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.databind.PropertyNamingStrategy;
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import com.fasterxml.jackson.databind.annotation.JsonNaming;
+import com.fasterxml.jackson.databind.annotation.JsonSerialize;
+import com.fasterxml.jackson.databind.ser.std.ToStringSerializer;
+
+import org.fossasia.openevent.app.common.data.models.dto.ObservableString;
+import org.fossasia.openevent.app.common.utils.json.ObservableStringDeserializer;
+import org.fossasia.openevent.app.common.utils.json.ObservableStringSerializer;
+
+import java.io.File;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+import lombok.NoArgsConstructor;
+
+@Data
+@Builder
+@AllArgsConstructor
+@NoArgsConstructor
+@JsonNaming(PropertyNamingStrategy.KebabCaseStrategy.class)
+@SuppressWarnings({ "PMD.TooManyFields" })
+@JsonInclude(JsonInclude.Include.ALWAYS)
+@EqualsAndHashCode(exclude = { "creating", "deleting" })
+@JsonAutoDetect(fieldVisibility = JsonAutoDetect.Visibility.ANY, getterVisibility = JsonAutoDetect.Visibility.NONE,
+    setterVisibility = JsonAutoDetect.Visibility.NONE, creatorVisibility = JsonAutoDetect.Visibility.NONE,
+    isGetterVisibility = JsonAutoDetect.Visibility.NONE)
+public class CreateEvent {
```

Comment:
We don't need another model. Original model can be reused

Reply:
Original model Event structure is different from the requested one, it contains many unnecessary fields and also I had to assign default values for all the fields. That's why I created another object.

Reply:
This is duplicating code, we'll have to maintain two models this way. Please use the existing one. Because having extra fields poses no problem in POST request. For default fields, one static method can be created to populate those when creating a new one

Reply:
@iamareebjamal   I've tried using the Event object already present but it contains id tag when JSON request is made which I'm not able to remove because it gives HTTP 400 error - `"Event id not found"` . Is there any way I can hide that id tag( `@JsonIgnore` doesn't work for it, neither does `@JsonInclude(JsonInclude.Include.NON_NULL` when setting id to null)  like currently it's being done in Ticket object. In the Ticket create request, the id tag is not present although it is present in the Object, i can't understand how it's done.

Reply:
Null fields should not be read by server. Can you please post the request being generated

Reply:
When id is set to null, the request doesn't generate. The event create activity crashes.

Reply:
id field is not setting to null by default. This is the request JSON -
 `D/OkHttp: {"data":{"type":"event","id":"0","attributes":{"analytics":{"checkedIn":{},"donationTickets":{},"freeTickets":{},"paidTickets":{},"soldDonationTickets":{},"soldFreeTickets":{},"soldPaidTickets":{},"totalAttendees":{},"totalSale":{},"totalTickets":{}},"bank-details":"example","can-pay-by-bank":"false","can-pay-by-cheque":"false","can-pay-by-paypal":"false","can-pay-by-stripe":"false","can-pay-onsite":"false","cheque-details":"example","code-of-conduct":"example","description":"example","ends-at":"2018-01-30T16:54:56.956+05:30","event-delegate":{"header":"LIVE","headerId":2337004},"external-event-url":"http://www.example.com","has-organizer-info":"false","is-complete":false,"is-sponsors-enabled":"false","latitude":"0.0","location-name":"example","logo-url":"http://www.example.com","longitude":"0.0","name":"I fgyhuh","onsite-details":"example","organizer-description":"example","organizer-name":"example","original-image-url":"http://www.example.com","payment-country":"example","payment-currency":"INR","paypal-email":"example","privacy":"example","searchable-location-name":"example","starts-at":"2018-01-24T16:54:56.956+05:30","state":"draft","ticket-url":"http://www.example.com","timezone":"Asia/Kolkata","is-map-shown":"false","is-sessions-speakers-enabled":"false","is-tax-enabled":"false","is-ticketing-enabled":"false"}}}`

Reply:
Set to null